apps/communications/contacts/js/utilities/object.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+(function(exports) {
+
+  exports = exports || {};
+
+  var object = {
+    // Checks if an object fields are empty, by empty means
+    // field is null and if it's an array it's length is 0
+    isEmpty: function (obj, fields) {
```

Comment:
`ContactService.isEmpty` checks if the DataBase is empty. This functions checks if an object (any) have empty fields, so I don't think putting it in ContactService is a good idea.

Plus, `ContactService` will be Thread.js in the near future, so no reason to add anything there.